__tests__/Place/place.test.ts

```diff
@@ -0,0 +1,108 @@
+import axios from "axios";
+import { Place, Rating } from "../../src/db/types";
+import { dbConnection } from "../../src/db/mongoConnection";
+
+describe("Place REST endpoints", () => {
+	it("/addPlace adds place to database", async () => {
+		const { _db, _connection } = await dbConnection();
+		await _db.dropDatabase();
+		await _connection.close();
+
+		let place!: Place;
+		let responseStatus!: number;
+		try {
+			const { data, status } = await axios.post<Place>("http://localhost:3030/addPlace", {
+				_id: "faketestid1",
+			});
+			place = data;
+			responseStatus = status;
+		} catch (e) {
+			console.log(e);
+		}
+
+		expect(responseStatus).toEqual(200);
+		expect(place).toMatchObject<Place>({
+			_id: "faketestid1",
+			avgBraille: null,
+			avgFontReadability: null,
+			avgGuideDogFriendly: null,
+			avgNavigability: null,
+			avgStaffHelpfulness: null,
+		});
+	});
+	it("/addPlace add several places", async () => {
+		const { _db, _connection } = await dbConnection();
+		await _db.dropDatabase();
+		await _connection.close();
+
+		let place!: Place;
+		let responseStatus!: number;
+		try {
+			const { data, status } = await axios.post<Place>("http://localhost:3030/addPlace", {
+				_id: "faketestid2",
+			});
+			place = data;
+			responseStatus = status;
+		} catch (e) {
+			console.log(e);
+		}
+
+		expect(responseStatus).toEqual(200);
+		expect(place).toMatchObject<Place>({
+			_id: "faketestid2",
+			avgBraille: null,
+			avgFontReadability: null,
+			avgGuideDogFriendly: null,
+			avgNavigability: null,
+			avgStaffHelpfulness: null,
+		});
+
+		let place2!: Place;
+		let responseStatus2!: number;
+		try {
+			const { data, status } = await axios.post<Place>("http://localhost:3030/addPlace", {
+				_id: "faketestid3",
+			});
+			place2 = data;
+			responseStatus2 = status;
+		} catch (e) {
+			console.log(e);
+		}
+
+		expect(responseStatus2).toEqual(200);
+		expect(place2).toMatchObject<Place>({
+			_id: "faketestid3",
+			avgBraille: null,
+			avgFontReadability: null,
+			avgGuideDogFriendly: null,
+			avgNavigability: null,
+			avgStaffHelpfulness: null,
+		});
+	});
+	it("/addPlace throws error when duplicate place is added", async () => {
+		let place!: Place;
+		try {
+			const { data } = await axios.post<Place>("http://localhost:3030/addPlace", {
+				_id: "faketestid3",
+			});
+			place = data;
+		} catch (e) {
+			expect(place).toBeUndefined();
+		}
```

Comment:
let's try and make the tests a bit more self-contained than this. currently, this test is relying on operations done from another test when it really doesn't have to. we should try and arrange the tests according to the Arrange-Act-Assert framework of test design. using this test as an example, here's how this test would go:

Arrange: drop existing database (done in beforeEach callback), post some place to the database
Act: post a place with the same id as the arrange step
Assert: expect the data returned from the post in the act step to be undefined

there are likely more tests than just this one that can be refactored to fit this design better.